sv_ttk/__init__.py

```diff
@@ -54,18 +54,15 @@ def get_theme(cls, root: tkinter.Misc | None = None) -> str:
     @classmethod
     def set_theme(cls, theme: str, root: tkinter.Misc | None = None) -> None:
         cls.load_theme(root)
-        if theme not in {"dark", "light"}:
+        if theme.lower() not in {"dark", "light"}: # Maybe someone will write Dark
             raise RuntimeError("not a valid sv_ttk theme name: {}".format(theme))
 
         cls.tcl.call("set_theme", theme)
 
     @classmethod
     def toggle_theme(cls, root: tkinter.Misc | None = None) -> None:
         cls.load_theme(root)
-        if cls.get_theme() == "light":
-            cls.set_theme("dark")
-        else:
-            cls.set_theme("light")
+        cls.set_theme("dark" if cls.get_theme() == "light" else "light")
```

Comment:
I don't really like this. It's shorter, but if expressions also make the code harder to read.

Reply:
I don't think it i harder to read. (Actually, at first I also think it is harder to read but Moosems advise me to use it. Later I can write and read it very well.) And I think it is the right syntax for the toggle function.

Reply:
But, if you still think it is hard to read, I can revert it.